build/preview

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eEuo pipefail
+
+SCRIPT_RELATIVE_DIR=$(dirname "${BASH_SOURCE[0]}")
+ROOT_RELATIVE_DIR=$(dirname "${SCRIPT_RELATIVE_DIR}")
+
+cd $ROOT_RELATIVE_DIR
+
+# get current version info
+VERSION=`git log -1 --format=%cd --date="format:%Y.%-m.%-d%H"`
+
+# Get existing name & description
+DISPLAY_NAME="$(jq -r .displayName package.json) Preview"
+DESCRIPTION="$(jq -r .description package.json) Preview"
```

Comment:
The description currently has brackets around `Preview`. Do we want to keep these?

<img width="413" alt="CleanShot 2021-11-09 at 14 21 40@2x" src="https://user-images.githubusercontent.com/45985/140931557-a4837214-21d6-4c95-8ea2-062a2ac951e5.png">

Reply:
Good catch, fixed!